config/routes.rb

```diff
@@ -15,8 +15,12 @@
 
   #Connections
   post '/api/classes/:id/request', to: 'api/connections#request_match', as: 'request_match'
-  put '/api/match_request/:id/', to: 'api/connections#update_match', as: 'update_match'
-  get '/api/notifications', to: 'api/connections#get_notifications', as: 'get_notifications'
+  put '/api/match_request/:id/', to: 'api/connections#update_match', as: 'update_match' # Answer or cancel match request
+  get '/api/notifications', to: 'api/connections#get_notifications', as: 'api_get_notifications'
+  put '/api/connections/:id/', to: 'api/connections#update_connection', as: 'update_connection' # End connection
+
+  #Ratings/reviews
+  post '/api/connections/:id/rate', to: 'api/ratings#rate_class', as: 'rate_class'
```

Comment:
Not relevant at this point but in case you don't know about it, you can use this to simplify some of the routes
https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default

Reply:
Yeah I read that before, but I didn't really like that for two reasons: 1. I can't explicitly tell what routes are being created or where they are routed to and 2. Creates extra routes that I might not need

Reply:
It's less explicit, yes. You can specify the routes you want with `only` and `except`